manifests/complete-postgres-manifest.yaml

```
@@ -107,9 +107,13 @@ spec:
     requests:
       cpu: 10m
       memory: 100Mi
+      hugepages-2Mi: 128Mi
+      hugepages-1Gi: 1Gi
     limits:
       cpu: 500m
       memory: 500Mi
+      hugepages-2Mi: 128Mi
+      hugepages-1Gi: 1Gi
```

```suggestion
      # hugepages-2Mi: 128Mi
      # hugepages-1Gi: 1Gi
    limits:
      cpu: 500m
      memory: 500Mi
      # hugepages-2Mi: 128Mi
      # hugepages-1Gi: 1Gi
```

lets comment these lines so that the e2e tests continue to work. These manifests are only here to show examples.